include/rocksdb/trace_reader_writer.h

```diff
@@ -36,6 +36,10 @@ class TraceReader {
 
   virtual Status Read(std::string* data) = 0;
   virtual Status Close() = 0;
+
+  // Allow to restart from the beginning such that Replayer can replay multiple
+  // times without being recreated.
```

Comment:
Indicate in the comment that the trace reader will seek back to the trace file header.

Reply:
Done